### ISSUE TITLE
User rights, deleting items

### ISSUE THREAD

**Issue by tammets** [centre-for-educational-technology/wateract#52]
Users are able to delete springs, observations and measurements:

**Registered user** is able to:

- delete spring as long it's **not confirmed**
- delete observations
- delete measurements

**Editor** is able to delete springs, measurements and observations that are in the same county as marked in user's profile.

**Comment by ailimadisson** [comment]
registered user can only delete as long spring / observation / measurement is in draft status.

**Comment by mvainu** [comment]
Senine praktiline kasutus on näidanud, et observation'ite puhul ei ole esitatud vaatluste muutmise/kustutamise mittelubamine põhjendatud, vaid pigem tekitab asjatuid ebamugavusi. Kuna toimetajad neid üle ei kontrolli, siis jääb kogu vastutus sisestajale. Kui keegi jätab näiteks kogemata trükivea sisse ja kunagi hiljem oma esitatud vaatlust vaatab, siis ei saa ta seda kuidagi parandada või esitab vaatluse lihtsalt prooviks, kuid ei saa seda siis hiljem kustutada. 

Seega võib Registered Useri õiguseid lihtsustada nii, et oleks:
- edit/delete own observations

Sama ka Editori puhul:
praeguse "edit /delete own observations as long it’s draft" asemel oleks
- edit/delete own observations

**Comment by ailimadisson** [comment]
Ok. Allowing to edit / delete own items even if not draft.

At the moment draft and submitted observations and measurements are basically same. Even draft observations and measurements are displayed for everyone. Maybe I should see draft ones only on my dashboard? Or what's the difference?

**Comment by mvainu** [comment]
Yes, it would be reasonable if draft observations and measurements would be visible only on My dashboard. For example, if the observer is in a hurry on the field and wants to write down only some measured numbers outside and save the observation as draft for later completion. As that observation form is clearly incomplete, then it would be logical, that is not be visible to others. If the observer completes the observation form later, and thinks that it is ok, then he/she submits it and it becomes public. 

The possibility to edit and delete submitted observations would become handy if the submitter realises later that he/she has made an error. For example, one can imagine a situation where the observer discovers that he/she has submitted the observation for a wrong spring (we have at least one such case already). Or he/she may discover a typo after reading through his/her observations at some later moment in time etc.

So, in short, saving as draft should indicate that the user recognises that the observation form is incomplete, therefore invisible to the public. Submitting should indicate that the user believes that the form is complete.
